### PR TITLE
Add ignore_cert option to ignore TLS/SSL certificate errors for self sig...

### PR DESCRIPTION
**Pull request by thet**
...ned certificates when using the ldaps uri schema.

### CODE REVIEW

**Comment by thet** [comment]
@rnixx @jensens @saily FYI

**Comment by thet** [comment]
@rnixx build is green

File: src/pas/plugins/ldap/properties.py [hunk 1]
```diff
@@ -202,6 +203,7 @@ def __init__(self, plugin):
 
     # XXX: Later
```
Comment (rnixx) [inline src/pas/plugins/ldap/properties.py]: (re)move XXX message

Comment (thet) [inline src/pas/plugins/ldap/properties.py]: my ``ignore_cert`` changes are actually not related to the ``start_tls`` feature. but looking at https://github.com/bluedynamics/node.ext.ldap/blob/master/src/node/ext/ldap/base.py#L124 it actually looks like the ``start_tls`` feature is implemented, but just not exposed in the controlpanel.
what do do?

**Comment by thet** [comment]
Looks like beeing obsolete due to https://github.com/collective/pas.plugins.ldap/commit/5479b898ad909473b1b08195d39c24bc8399f0bd - no?
/cc @rnixx

**Comment by jensens** [comment]
yes
